chainer/training/extensions/plot_report.py

```diff
@@ -50,6 +47,22 @@ class PlotReport(extension.Extension):
     It also adds ``'epoch'`` and ``'iteration'`` entries to each result
     dictionary, which are the epoch and iteration counts at the output.
 
+    .. warning::
+
+        If your environment needs to specify a backend of matplotlib
+        explicitly, please call ``matplotlib.use`` before importing Chainer.
+        For example:
+
+        .. code-block:: python
+
+            import matplotlib
+            matplotlib.use('Agg')
+
+            import chainer
+
+        Then, once ``chainer.training.extensions`` is imported,
+        ``matplotlib.use`` will never be available.
```

Comment:
I think "``matplotlib.use`` will have no effect" is more appropriate because ``matplotlib.use`` does be available in the sense that we can call the function anyway.  Also, could you add the link to the corresponding part of the document of matplotlib [here](http://matplotlib.org/faq/usage_faq.html#what-is-a-backend) ?